Competition2017/src/competition/subsystems/drive/DriveSubsystem.java

```diff
@@ -64,9 +64,9 @@ public DriveSubsystem(WPIFactory factory, XPropertyManager propManager, ShiftSub
         leftDrive.createTelemetryProperties("Left primary", propManager);
         leftDriveFollower.createTelemetryProperties("Left follower", propManager);
         
-        this.rightDrive = factory.getCANTalonSpeedController(21);
+        this.rightDrive = factory.getCANTalonSpeedController(20);
```

Comment:
This would be an electrical change - since the encoder is only read on the master.

That being said, at the very least, we need to document this.